lib/contract/jsvm/executor_test.go

```diff
@@ -3,33 +3,48 @@ package jsvm
 import (
 	"testing"
 
-	"github.com/magiconair/properties/assert"
-
-	"boscoin.io/sebak/lib/contract/api"
 	"boscoin.io/sebak/lib/contract/context"
 	"boscoin.io/sebak/lib/contract/payload"
+	"boscoin.io/sebak/lib/contract/test"
+	"github.com/stretchr/testify/assert"
 )
 
-func TestExecutor(t *testing.T) {
+func Test_JSVM_Executor(t *testing.T) {
+	testAddress := "testadress"
+	testCode := `
+function Hello(helloarg){
+    return HelloWorld(helloarg)
+}
+`
+
 	deployCode := &payload.DeployCode{
 		ContractAddress: testAddress,
 		Code:            []byte(testCode),
 		Type:            payload.JavaScript,
 	}
-	testStateClone.PutDeployCode(deployCode)
 
-	context := &context.Context{
-		StateStore: testStateStore,
-	}
-	api := api.NewAPI(context, testAddress, nil)
-	ex := NewOttoExecutor(context, api, deployCode)
+	ctx := test.NewMockContext(testAddress, nil)
+	api := test.NewMockAPI(ctx, testAddress)
+	api.SetHelloworldFunc(func(ctx context.Context, greeting string) (string, error) {
```

Comment:
`context.Context` and `api.API` are interface, So testing within `contract/jsvm` or another places in `contract`,  We can use  `contract/test/MockAPI` and `contract/test/MockContext`.  Their was not complete for now, but this test works well with this Mock struct.